network/monuments.csv

```diff
@@ -92,6 +92,7 @@ MCNL,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.507,Stainless Steel Rods,10
 METH,50251M001,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.56,Stainless Steel Rods,6,2010-08-01T00:00:00Z,9999-01-01T00:00:00Z,,
 MING,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.54,Stainless Steel Rods,10,2010-04-28T00:00:00Z,9999-01-01T00:00:00Z,,
 MKNO,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.551,Stainless Steel Rods,10,2011-05-11T00:00:00Z,9999-01-01T00:00:00Z,,
+MKP1,,Forced Centering,Steel Mast,-2,Ground Screw,10,2025-03-17T20:32:38Z,9999-01-01T00:00:00Z,Sandstone,Landslide deposit
```

Comment:
The depth of the ground screw should be `0.5` not `10`. 
Normally we don't put hours, minutes and seconds in the timestamp in this table.